.changeset/olive-bags-drive.md

```diff
@@ -0,0 +1,18 @@
+---
+"@astrojs/solid-js": minor
+---
+
+Adds a `devtools` option
+
+Enabling `devtools` will enable the [official Solid Devtools](https://github.com/thetarnav/solid-devtools) in development:
```

Comment:
```suggestion
You can enable the [official Solid Devtools](https://github.com/thetarnav/solid-devtools) while working in development mode by setting `devtools:true` in your `solid()` integration config:
```

Reply:
Thanks! We'll do the since thing don't worry